Cargo.toml

```diff
@@ -3,6 +3,7 @@ resolver = "2"
 members = [
     "devenv",
     "devenv-run-tests",
+    "xtask",
```

Comment:
Why not `devenv-generate-manpages`?

Reply:
Sort of a convention https://github.com/matklad/cargo-xtask?tab=readme-ov-file. You could add other tasks to this crate to avoid slowing down builds of the main package.

Reply:
Cargo has separate crates for each task and then sets aliases for them in `.cargo/config.toml`. Others use a single crate with subcommands. Same thing.